src/config.js

```diff
@@ -56,7 +56,7 @@ const gaTrackingId = () => {
 const learningpathFrontendDomain = () => {
   switch (ndlaEnvironment) {
     case 'local':
-      return 'http://localhost:30017';
+      return 'http://localhost:30008';
```

Comment:
Kvifor har du gjort dette?

Reply:
Fant ut at vi trodde denne var feil, men vi bruker kanskje ndla-frontend istedet for learningpath-frontend for linkene her nå?

Reply:
Tulla, da måtte vi jo oppdatert de andre miljøene også. Så denne skal endres for lokalt, men ser nå at det skal være `30007`